src/modules/quizzes.js

```diff
@@ -111,15 +117,15 @@ class Quizzes {
    *    versionId: '123'
```

Comment:
Let's update the example too 👍 `versionId` => `quizVersionId`

Reply:
I remember specifically searching for `versionId` and somehow missed this 🤦